### PR TITLE
network: Allow vxlan to be created without .network file

### PR DESCRIPTION
**Pull request by ssahani**
closes https://github.com/systemd/systemd/issues/17064

### CODE REVIEW

File: man/systemd.netdev.xml [hunk 1]
```diff
@@ -819,6 +819,13 @@
           <para>Accepts the same key in [VXLAN] section.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>Independent=</varname></term>
+        <listitem>
+          <para>Takes a boolean. When true vxlan does not require .network file. Defaults to <literal>false</literal>.
```
Comment (poettering) [inline man/systemd.netdev.xml]: I find this confusing, i.e. the primary effect of this is that the netdev is not associated with an underlying iface, and that's what should be mentioned here, no?

**Review by poettering — CHANGES_REQUESTED** [review]

**Comment by yuwata** [comment]
@ssahani I've updated man page as requested by @poettering, and added a test for the setting.

@poettering PTAL.

**Comment by ssahani** [comment]
Thank you @yuwata

**Comment by yuwata** [comment]
CentOS CI (Arch Linux + sanitizers) failure is in `test_wireguard()`, seems not related.

**Pull request merged**